go/store/prolly/node_store.go

```diff
@@ -32,12 +32,11 @@ const (
 
 // NodeStore reads and writes prolly tree Nodes.
 type NodeStore interface {
+	// Read reads a prolly tree mapNode from the store.
+	Read(ctx context.Context, ref hash.Hash) (mapNode, error)
```

Comment:
How do people outside this package using this public interface use this private type (mapNode)?

Reply:
couldn't remember why `Node` was public, this is it